### DISCUSSION TITLE
Behaviour of speakerpanel, iOS

### DISCUSSION THREAD

**Discussion by NWSS** [music-assistant/support#1066] [Feature requests and ideas]
At the moment trying to adjust the volume of a speaker is a bit tricky. This is because the panel also reacts on the slide gesture to pull it back in. So when I try to adjust the volume, I slide back the panel instead of adjusting the volume. Some suggestions:

1. Don't let slide touches retrieve or disappear the panel, only taps AND/OR
2. Introduce volume buttons UP and DOWN with increments of 5% (or adjustable increments in a config-file) 

![Schermafbeelding 2023-01-13 om 08 51 04](https://user-images.githubusercontent.com/121284555/212268905-57ad7650-04cb-4d0c-86b5-5faf7cfa94fd.png)

**Comment by OzGav** [comment]
relates to v1
